tools/mkmapdata/mkmapdata.sh

```diff
@@ -288,7 +280,7 @@ else
 fi
 
 # Set timestamp for names below 
-CDATE="c"`date +%y%m%d`
+CDATE="c"`date -d "0 days" +%y%m%d`
```

Comment:
See comment about this in the other PR. Is this for setting of daylight savings time? If so I especially think we shouldn't mess with it.

Reply:
Adding comment explaining the use of the `date -d` flag, as discussed [here](https://github.com/ESCOMP/CTSM/pull/823/files#r359095927).